drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -47,71 +51,59 @@ static int bq274xx_command_reg_read(const struct device *dev, uint8_t reg_addr,
 	return 0;
 }
 
-static int bq274xx_control_reg_write(const struct device *dev,
+static int ctrl_reg_write16(const struct device *dev,
 				     uint16_t subcommand)
 {
 	const struct bq274xx_config *config = dev->config;
-	uint8_t i2c_data, reg_addr;
-	int status = 0;
+	uint8_t i2c_data;
+	int ret;
 
-	reg_addr = BQ274XX_COMMAND_CONTROL_LOW;
 	i2c_data = (uint8_t)((subcommand)&0x00FF);
 
-	status = i2c_reg_write_byte_dt(&config->i2c, reg_addr,
+	ret = i2c_reg_write_byte_dt(&config->i2c, BQ274XX_CMD_CTRL_LOW,
 				       i2c_data);
-	if (status < 0) {
-		LOG_ERR("Failed to write into control low register");
+	if (ret < 0) {
 		return -EIO;
 	}
 
 	k_msleep(BQ274XX_SUBCLASS_DELAY);
 
-	reg_addr = BQ274XX_COMMAND_CONTROL_HIGH;
 	i2c_data = (uint8_t)((subcommand >> 8) & 0x00FF);
 
-	status = i2c_reg_write_byte_dt(&config->i2c, reg_addr,
+	ret = i2c_reg_write_byte_dt(&config->i2c, BQ274XX_CMD_CTRL_HIGH,
 				       i2c_data);
-	if (status < 0) {
-		LOG_ERR("Failed to write into control high register");
+	if (ret < 0) {
 		return -EIO;
 	}
 
 	return 0;
 }
 
-static int bq274xx_command_reg_write(const struct device *dev, uint8_t command,
-				     uint8_t data)
+static int cmd_reg_write8(const struct device *dev, uint8_t command,
+				     uint8_t val)
```

Comment:
Please fix indentations (applies to the entire file)
```suggestion
static int cmd_reg_write8(const struct device *dev, uint8_t command,
			  uint8_t val)
```